pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -136,8 +136,12 @@ func validatePlacementPolicy(policy *placementv1beta1.PlacementPolicy) error {
 			allErr = append(allErr, err)
 		}
 	}
-	if err := validateTolerations(policy.Tolerations); err != nil {
-		allErr = append(allErr, err)
+	if policy.PlacementType != placementv1beta1.PickFixedPlacementType {
```

Comment:
suggest to call the validateTolerations in the switch case to make the code clean